### PR TITLE
Navbar links

### PR DESCRIPTION
**Pull request by Robbie-Smith**
@case-eee @kheppenstall @jdconrad89 @Dpalazzari @drod1000 

I added tests for the navbar and updated routes to reflect what happens when you are actually using kickstarter. I also added the links to the navbar, and updated the styling on the user-new form.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -45,4 +36,14 @@ def oauth_user_data
   def with_facebook?
     !oauth_data.nil?
   end
+
+  def successful_login
+    session[:user_id] = @user.id
+    flash[:success] = "You are successfully logged in"
```
Comment (Dpalazzari) [inline app/controllers/sessions_controller.rb]: want to change to :

`redirect_to profile_path, success: "You are successfully logged in"`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -30,6 +44,11 @@ def destroy
   private
 
   def user_params
-    params.require(:user).permit(:name, :phone_number, :email, :password)
+    params.require(:user).permit(:name, :avatar, :biography, :phone_number, :email, :password)
   end
+
+  def found_user
+    User.find(session[:user_id])
+  end
```
Comment (Dpalazzari) [inline app/controllers/users_controller.rb]: would a before_action make more sense here?

Comment (Robbie-Smith) [inline app/controllers/users_controller.rb]: @Dpalazzari No, we were just using that argument on line 51 in a few places, so I just pulled it out into a method.

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (Robbie-Smith) [inline app/controllers/sessions_controller.rb]: @Dpalazzari Yea, good catch.

**Comment by Robbie-Smith** [comment]
@Dpalazzari I made those changes, if you're cool with them. Please merge.

**Comment by jdconrad89** [comment]
@Robbie-Smith navbar looks great! thanks for making those changes

**Pull request merged**